### DISCUSSION TITLE
Add image or text display type for SettingInputType of App development

### DISCUSSION THREAD

**Discussion by garryxiao** [datalust/seq-tickets#1151] [Ideas]
SettingInputType.Image - Display an image in the setting UI page. For example a dynamic generated QR code for scan to identify user.

**Comment by garryxiao** [comment]
Also found "Display Name" and "HelpText" in Chinese will not be displayed correctly.

**Reply by nblumhardt** [reply]
Thanks for the note, we'll check this out! :+1:

**Reply by Anton-Nikishkin** [reply]
> Also found "Display Name" and "HelpText" in Chinese will not be displayed correctly.

Russian language not diasplayed too.

**Reply by nblumhardt** [reply]
Thanks for the extra info, @Anton-Nikishkin .

All,

I'm having trouble reproducing this; I updated my copy of the "File Archive" app with a Chinese hello-world description:

![image](https://user-images.githubusercontent.com/342712/113453424-453bc300-9449-11eb-8bd7-416e86c8dec7.png)

But Seq loads this up and seems to display it just fine:

![image](https://user-images.githubusercontent.com/342712/113453475-63092800-9449-11eb-853a-162df00faf94.png)

Am I misunderstanding where the issue lies, or do you have any ideas what else might cause this?

Thanks again!

**Reply by Anton-Nikishkin** [reply]
![image](https://user-images.githubusercontent.com/8031277/114679470-d8f87380-9d14-11eb-8f26-48d33e2768f0.png)
![image](https://user-images.githubusercontent.com/8031277/114679546-eada1680-9d14-11eb-9ef3-cfb3802c383d.png)

**Reply by Anton-Nikishkin** [reply]
Sorry, I mean SeqApp Name and Description

**Reply by nblumhardt** [reply]
Thanks, will take another look :+1:

One thing to check out, [is your `.cs` source file UTF-8 encoded](https://stackoverflow.com/questions/840065/how-to-change-source-file-encoding-in-csharp-project-visual-studio-msbuild-ma/53259212)?

**Reply by Anton-Nikishkin** [reply]
![image](https://user-images.githubusercontent.com/8031277/115197539-c48fee80-a0f9-11eb-9672-3fcb8cb2e688.png)
Yes, it is!

**Comment by garryxiao** [comment]
I am sure it is.Regards,Garry Xiao-------- Original message --------From: Nicholas Blumhardt ***@***.***>Date: Thu, Apr 15, 2021, 10:11 AMTo: datalust/seq-tickets ***@***.***>Cc: Garry Xiao ***@***.***>, Author ***@***.***>Subject: Re: [datalust/seq-tickets] Add image or text display type for SettingInputType of App development (#1151)

Thanks, will take another look 👍

One thing to check out, is your .cs source file UTF-8 encoded?
